lib/state/settings/reducer.js

```diff
@@ -57,12 +67,28 @@ const markdownEnabled = ( state = false, action ) => {
 	return action.markdownEnabled;
 };
 
+const visiblePanes = ( state = [
+	'navigation-bar',
+	'note-list',
+	'note-editor',
+], action ) => {
+	if ( SHOW_PANE !== action.type && HIDE_PANE !== action.type ) {
+		return state;
+	}
```

Comment:
this is not needed since `get()` defaults to the existing state; it is redundant.